docs/Catalog/Packager.md

```diff
@@ -0,0 +1,182 @@
+
+### Workflow Run RO-Crate
+
+When enabled, this will create a package from any folder containing an
+`ro-crate-manifest.json`. [RO-Crate](https://www.researchobject.org/ro-crate/)
+is a metadata standard for describing research data.  The Workflow Run working
+group adds three additional profiles, which are supported in the latest versions of
+[nf-prov](https://github.com/nextflow-io/nf-prov). You need to explicitly
+configure it to use wrroc, using a `nextflow.config` file [like this](https://github.com/famosab/wrrocmetatest):
+
+```groovy
+plugins {
+ id 'nf-prov@1.4.0'
+}
+
+prov {
+ enabled = true
+ formats {
+   wrroc {
+     file = "${params.outdir}/ro-crate-metadata.json"
+     overwrite = true
+     agent {
+       name = "John Doe"
+       orcid = "https://orcid.org/0000-0000-0000-0000"
+     }
+      license = "https://spdx.org/licenses/MIT"
+      profile = "provenance_run_crate"
+   }
+ }
+}
+```
+
+Note that Research Objects identify people using an ORCID iD, which anyone can
+get for free at [the ORDiD website](https://orcid.org/).
+
+The package will be created in the same bucket as the `outdir`, using the
+last two path components as the package name. If there are fewer than two
+components, it will use a default prefix or suffix.
+
+## SQS Message Processing
+
+The primary interface to the Auto-Packaging is through the
+SQS queue in the same account and region as your Quilt stack,
+listed in `PackagerQueue` under the Outputs. The queue URL will look something like:
+
+```url
+https://sqs.us-east-1.amazonaws.com/XXXXXXXXXXXX/PackagerQueue-XXXXXXXXXXXX
+```
+
+The body of the message is the stringified JSON of a package description.
+There is only one required parameter:
+
+```json
+{
+  "source_prefix": "s3://data_bucket/source/folder/metadata.json"
+}
+```
+
+This will create a package in the same bucket as the source folder, with the
+package name being the last two components of the source URI (e.g., `source/folder`).
+By default, it will assume the last component is a metadata file, and
+try to parse that as json.
+If you are specifying a folder, you should instead end the URI with a `/`.
+
+Optionally, you can control the package name, metadata, and other settings by
+explicitly specifying any of the following fields:
+
+```jsonc
+{
+  "source_prefix": "s3://data_bucket/source/folder/",
+  "registry": "package_bucket",  // may be the same as `data_bucket`
+  "package_name": "prefix/suffix",
+  "metadata": { "key": "value" },  // dictionary
+  "metadata_uri": "metadata.json", // alternative to `metadata`, relative or absolute
+  "message": "Commit message for the package revision", // string
+  "workflow": "alpha", // name of a valid metadata workflow
+  "should_copy": false // boolean
+  // false = point to data in the source bucket (default)
+  // true = copy data to the package bucket
+}
+```
+
+## Custom EventBridge Rules
+
+EventBridge event in your account (from any bus, in any region) into a
```

Comment:
actually we need to check if EventBridge can send to SQS cross-region

Reply:
Are you going to check that, or should I?

Reply:
AFAICT it can't
I'm not sure we should mention since it's AWS limitation